README.md

```diff
@@ -235,3 +235,66 @@ This concludes the flow and you should be able to see the secret configured in t
 ```
 kubectl get secret token-secret -o yaml
 ```
+
+
+### SPI Logging concept.
+
+It is important to keep consistency between components in terms of the ways and form of how
+the events are logged. It makes the development and support of these components much easier.
+
+Since SPI is OpenShift/Kubernetes oriented framework we are inspired by [Operator SDK Logging](
+https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/) and [structured logging](
+https://www.client9.com/structured-logging-in-golang/) methods. That means usage of [logr](https://pkg.go.dev/github.com/go-logr/logr) interface
+to log and [zap](https://pkg.go.dev/github.com/go-logr/zapr) as a backend.
+
+
+#### How to use
+```
+func (r *SPIAccessTokenReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+....
+	lg := log.FromContext(ctx)
+	lg.Info("Reconciling")
+....
+}	
+```
+```
+func CallbackErrorHandler(w http.ResponseWriter, r *http.Request) {
+	lg := log.FromContext(r.Context())
+	lg.Info("CallbackErrorHandler")
+	....
+}
+```
+#### Log levels
+INFO inclide
```

Comment:
is there a concept of INFO and DEBUG levels? I think there are only numbered levels, with lower the number, higher the priority.